generator/ServiceClientGeneratorLib/GeneratorDriver.cs

```diff
@@ -871,8 +871,7 @@ public static void UpdateNuGetPackagesInReadme(GenerationManifest manifest, Gene
             var nugetPackages = new Dictionary<string, string>();
             foreach (var service in manifest.ServiceConfigurations.OrderBy(x => x.ClassName))
             {
-                // Service like DynamoDB streams are included in a parent service.
-                if (service.ParentConfig != null || service.IsTestService)
```

Comment:
Yup ddbstreams is the only service using parent-base-name. I ran the generator just to be sure, but yeah if a service were to use it in the future this would break them.

I'm okay with just being safe for now and removing the comment but reverting the if statement